### ISSUE TITLE
Possible slight misuse of MinMax scaler

### ISSUE THREAD

**Issue by boborbt** [ml-unito/fair-networks#11]
How should the MinMax scaler be used at test time? The lines below seem to show that one would use the same scaling in the test test as the one used in the training set. Probably one would instead use a scaling that is fitted on the test set.

https://github.com/ml-unito/fair-networks/blob/1844180e0f3a8cb7b903627b4e5bfc880226753e/packages/fair/datasets/yale_b_dataset.py#L119-L121
